MinimedKit/Extensions/NSData.swift

```diff
@@ -71,6 +71,14 @@ extension Data {
      */
 }
 
+extension Data {
+    func reverseBytes() -> Data {
```

Comment:
You can use Data(data.reversed()) to create a new reversed data object.  There may be ways to work with RandomAccessCollection<Data> types, and avoid the copy, but I wasn't able to get that working in my swift playground.